src/lib/state.ts

```diff
@@ -97,19 +97,19 @@ interface NodeStateSchema0 {
   status: ZWaveNode["status"];
   ready: ZWaveNode["ready"];
   isListening: ZWaveNode["isListening"];
-  isFrequentListening: ZWaveNode["isFrequentListening"];
+  isFrequentListening: boolean;
   isRouting: ZWaveNode["isRouting"];
-  maxBaudRate: ZWaveNode["maxBaudRate"];
+  maxBaudRate: ZWaveNode["maxDataRate"];
   isSecure: ZWaveNode["isSecure"];
-  version: ZWaveNode["version"];
-  isBeaming: ZWaveNode["isBeaming"];
+  version?: number;
+  isBeaming: ZWaveNode["supportsBeaming"];
```

Comment:
If we do this, we should make sure we require at least version 7 as peer dependency and we should mark that in our readme too.

Reply:
the `version` property here references the protocol version that a node supports. I think you are talking about `node-zwave-js` right?

Reply:
I meant that the types don't work when we install z-wavejs 6, but I guess that doesn't matter because they fail anyway in that case.

Reply:
Actually, we should probably write out ALL types (in a new PR), so it's easy to spot type errors when we update versions, so we know what needs to be made compatible.

Reply:
oh I see, yeah we can do that in a separate PR